charts/iter8lib/templates/_k-spec-secret-istio.tpl

```diff
@@ -0,0 +1,12 @@
+{{- define "k.spec.secret-istio" -}}
+{{- $name := printf "%v-%v" .Release.Name .Release.Revision -}}
```

Comment:
@sriumcp perhaps you can comment on the elements that are essential from `_k-spec-secret.tpl`. `name` and `annotations`?